### PR TITLE
Hide mute button when moderator join via audio in breakout room.

### LINKED ISSUE #6941: Breakout rooms - Mic icon remains for moderator in main room
When a moderator joins audio only in a breakout room their mic icon in the main / parent meeting room remains and looks like it can be engaged to mute:

https://www.youtube.com/watch?v=eS3DtQfoinE&feature=youtu.be

The mic controls should be removed from the parent meeting room when the moderator joins audio in a breakout room.

### PR DESCRIPTION
**Pull request by Tainan404**
Closes #6941.

### CODE REVIEW

File: bigbluebutton-html5/imports/api/voice-users/server/publishers.js [hunk 1]
```diff
@@ -11,7 +11,7 @@ function voiceUser(credentials) {
 
   Logger.debug(`Publishing Voice User for ${meetingId} ${requesterUserId}`);
 
-  return VoiceUsers.find({ meetingId });
+  return VoiceUsers.find({ intId: requesterUserId });
```
Comment (antobinary) [inline bigbluebutton-html5/imports/api/voice-users/server/publishers.js]: Are you trying to have access to the voice user from main room in the breakout?

Comment (Tainan404) [reply bigbluebutton-html5/imports/api/voice-users/server/publishers.js]: Yes

Comment (capilkey) [reply bigbluebutton-html5/imports/api/voice-users/server/publishers.js]: Can you not just check to see if the person is in the current voice conference? When they join the breakout room's voice bridge they leave the main voice bridge.

Comment (Tainan404) [reply bigbluebutton-html5/imports/api/voice-users/server/publishers.js]: I don't have idea how to do this, could you give me an insight in how to do this in `HTML5`?

Comment (capilkey) [reply bigbluebutton-html5/imports/api/voice-users/server/publishers.js]: Publish VoiceUsers who are both in `meetingId` and have an `intId = requesterUserId`. If you get a result then the user is in the current meeting's voiceBridge and the mute button should be visible.

Comment (Tainan404) [reply bigbluebutton-html5/imports/api/voice-users/server/publishers.js]: But the old connections items aren't removed, so sometimes the register returned isn't the last, I tried remove, but it bugged the echo test. I added the updadeTime to make sure that I'm getting the last item and comparing with meeting voice conf

Comment (capilkey) [reply bigbluebutton-html5/imports/api/voice-users/server/publishers.js]: I'm not quite sure what you mean when you say that the item isn't the last. The collections already update when you transfer the audio to the breakout room. I just tried it on my 2.2 server and as soon as the moderator joins the breakout room audio their microphone icon in the user list is removed.

**Pull request merged**